examples/java/build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
     dependencies {
-        classpath "org.jetbrains.kotlinx:kotlinx-benchmark-plugin:0.4.8"
+        classpath "org.jetbrains.kotlinx:kotlinx-benchmark-plugin:0.4.255-SNAPSHOT"
```

Comment:
255 is rather arbitrary here. In the main Kotlin repository we use this value because it's the maximum value a Kotlin component version can have, but here we have no such limitation.
Maybe it would be better to use an approach of other libraries, i.e. as soon as we released a version 0.4.x, we advance the version to 0.4.(x+1)-SNAPSHOT? Or, to avoid changing the version in examples after an each release, immediately to 0.5.0-SNAPSHOT?

Reply:
`0.5.0-SNAPSHOT` is now used. Release version is requested when a TeamCity release build is run.